mock-core-configs/etc/mock/templates/centos-7.tpl

```diff
@@ -7,8 +7,6 @@ config_opts['releasever'] = '7'
 config_opts['bootstrap_image'] = 'quay.io/centos/centos:7'
 config_opts['package_manager'] = 'yum'
 
-config_opts['yum_install_command'] += "{% if target_arch in ['x86_64', 'ppc64le', 'aarch64'] %} --disablerepo=centos-sclo*{% endif %}"
-
```

Comment:
This also breaks EPEL 7 builds, since we do need this to ensure SCLo content is only used at the right stage for EPEL 7 builds.

Reply:
I will adjust this, but can you please provide a reproducer? I have tested on multiple host/target combinations and I cannot see where this breaks.